src/main/java/frc/trigon/robot/subsystems/shooter/ShooterConstants.java

```diff
@@ -0,0 +1,120 @@
+import frc.trigon.robot.constants.RobotConstants;
+import org.trigon.hardware.RobotHardwareStats;
+import org.trigon.hardware.phoenix6.talonfx.TalonFXMotor;
+import org.trigon.hardware.phoenix6.talonfx.TalonFXSignal;
+import org.trigon.hardware.simulation.FlywheelSimulation;
+import org.trigon.utilities.mechanisms.SpeedMechanism2d;
+
+public class ShooterConstants {
+    private static final int
+            RIGHT_MOTOR_ID = 0,
+            LEFT_MOTOR_ID = 1;
+    private static final String
+            RIGHT_MOTOR_NAME = "rightShootingMotor",
+            LEFT_MOTOR_NAME = "leftShootingMotor";
+    static final TalonFXMotor
+            RIGHT_MOTOR = new TalonFXMotor(RIGHT_MOTOR_ID, RIGHT_MOTOR_NAME, RobotConstants.CANIVORE_NAME),
+            LEFT_MOTOR = new TalonFXMotor(LEFT_MOTOR_ID, LEFT_MOTOR_NAME, RobotConstants.CANIVORE_NAME);
+    private static final InvertedValue
+            RIGHT_MOTOR_INVERTED_VALUE = InvertedValue.Clockwise_Positive,
+            LEFT_MOTOR_INVERTED_VALUE = InvertedValue.CounterClockwise_Positive;
+    private static final NeutralModeValue NEUTRAL_MODE_VALUE = NeutralModeValue.Coast;
+    private static final double
+            P = RobotHardwareStats.isSimulation() ? 1 : 1,
+            I = RobotHardwareStats.isSimulation() ? 0 : 0,
+            D = RobotHardwareStats.isSimulation() ? 0 : 0,
+            KS = RobotHardwareStats.isSimulation() ? 0 : 0,
+            KV = RobotHardwareStats.isSimulation() ? 0 : 0,
+            KA = RobotHardwareStats.isSimulation() ? 0 : 0,
+            KG = RobotHardwareStats.isSimulation() ? 0 : 0;
+    private static final double GEAR_RATIO = 1;
+    static final boolean FOC_ENABLED = true;
+
+    private static final int
+            RIGHT_MOTOR_AMOUNT = 1,
+            LEFT_MOTOR_AMOUNT = 1;
+    private static final DCMotor
+            RIGHT_GEARBOX = DCMotor.getFalcon500Foc(RIGHT_MOTOR_AMOUNT),
+            LEFT_GEARBOX = DCMotor.getFalcon500Foc(LEFT_MOTOR_AMOUNT);
+    private static final double MOMENT_OF_INERTIA = 0.003;
+    static final FlywheelSimulation
+            RIGHT_SIMULATION = new FlywheelSimulation(RIGHT_GEARBOX, GEAR_RATIO, MOMENT_OF_INERTIA),
+            LEFT_SIMULATION = new FlywheelSimulation(LEFT_GEARBOX, GEAR_RATIO, MOMENT_OF_INERTIA);
+
+    static final SysIdRoutine.Config SYSID_CONFIG = new SysIdRoutine.Config(
+            Units.Volts.of(0.25).per(Units.Second),
+            Units.Volts.of(7),
+            Units.Second.of(1000000000)
+    );
+
+    private static final double MAX_DISPLAYABLE_VELOCITY = 5;
+    static final SpeedMechanism2d
+            RIGHT_MECHANISM = new SpeedMechanism2d("RightShooterMechanism", MAX_DISPLAYABLE_VELOCITY),
+            LEFT_MECHANISM = new SpeedMechanism2d("LeftShooterMechanism", MAX_DISPLAYABLE_VELOCITY);
+
+    static final double WHEEL_DIAMETER_METERS = 0;
+    public static final double ROTATIONS_TO_METERS = GEAR_RATIO / (WHEEL_DIAMETER_METERS * Math.PI);
```

Comment:
Maybe move this calculation into a separate method? Also maybe rename to ROTATIONS_TO_METERS_RATIO? Or something like that to make it more understandable.

Reply:
This is copied code from Triumph. I'll ask Yishai about it in person

Reply:
This calc shouldn't really be in a method since we want to calculate once. It could be transferred into one, but I don't see high value in that.
As for the name, the word "ratio" conveys how there's some number between one thing _to_ another. Now, instead of writing ratio, we're writing "rotations _to_ meters". It's like writing 'motor _to_ system". Maybe a better name would be `SHOOTER_ROTATIONS_TO_METERS`